tracker/repository/session_repository.py

```diff
@@ -0,0 +1,39 @@
+from abc import ABC
+from abc import abstractmethod
+from typing import Sequence
+
+from tracker.models import WorkoutSession
+
+from .base import CRUDRepositorySQLAlchemy
+
+
+class AbstractWorkoutSessionRepository(ABC):
```

Comment:
Why do you use `WorkoutSessionService` and `WorkoutSessionServiceImpl` in services, but here you use `AbstractWorkoutSessionRepository` and `WorkoutSessionRepository`? Decide, this should be consistent.